src/app/help/keyboard-shortcuts/keyboard-shortcuts.service.ts

```diff
@@ -0,0 +1,252 @@
+import { Injectable } from '@angular/core';
```

Comment:
Actually i used it in my previous internship which is a private repository, but 6 months back i took reference from some website tutorial for this, i'll look for it again and will share with you if i could find it.

Reply:
Were you able to locate the source? I need to study this file before we could merge, I'll be able to do it faster if you could locate it otherwise it might take some time and you'll have to be patient with me.

Reply:
Sorry for the late response, found it https://www.bennadel.com/blog/3382-handling-global-keyboard-shortcuts-using-priority-and-terminality-in-angular-5-0-5.htm , also i removed the unnecesary Unlisten code, as i went through the tutorial again, now the code has only essential parts. The unlisten was creating more confusion than good so removed that as well.

Reply:
I'll get back to you once I go through this.